src/UnitySampleProject/Assets/Samples/web3.unity SDK/2.5.0/Web3.Unity Samples/Scripts/Scenes/SampleMain/SygmaClient.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.Numerics;
+using System.Threading.Tasks;
+using ChainSafe.Gaming.Evm.Contracts;
+using ChainSafe.Gaming.Evm.Transactions;
+using ChainSafe.Gaming.SygmaClient;
+using ChainSafe.Gaming.SygmaClient.Dto;
+using ChainSafe.Gaming.SygmaClient.Types;
+using ChainSafe.Gaming.UnityPackage;
+using ChainSafe.Gaming.Web3;
+using Nethereum.Hex.HexTypes;
+using Scripts.EVM.Token;
+using UnityEngine;
+using UnityEngine.UI;
+using Environment = ChainSafe.Gaming.SygmaClient.Types.Environment;
+
+[Serializable]
+public class SygmaConfig
+{
+    [field: SerializeField] public ResourceType ResourceType { get; private set; } = ResourceType.Erc1155;
+    [field: SerializeField] public string TokenId { get; private set; } = "1";
+    [field: SerializeField] public Environment SygmaEnvironment { get; private set; } = Environment.Testnet;
+    [field: SerializeField] public uint DestinationChainId { get; private set; } = 338;
+    [field: SerializeField] public uint Amount { get; private set; } = 1;
+}
+
+public class SygmaClient : MonoBehaviour
```

Comment:
I would recommend renaming it to avoid confusion around names. This is more like SygmaSample that Client.